Task_Position_the_Cubes/index.css

```diff
@@ -1,46 +1,58 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
-.parent {
+.flexbox-container {
   width: 300px;
   height: 300px;
-  /* padding: 80px 50px;
-  padding-bottom: 80px;
-  padding: left 30px;;
-  border: 5px solid gray;
-  border-radius: 0px;*/
-  margin: 100px; 
-  background-color: #e42f68;
+  margin: 50%; 
+  background-color:blue;
+  flex-grow:3;
+  position: static;
+ 
 }
-.child-one {
-  background: red;
+.flexbox-item-1{
+background: #19c474;
 width: 100PX;
```

Comment:
The 'position: relative;' property is unnecessary since the default value for position is already static.